.ci-prepare.sh

```diff
@@ -6,7 +6,3 @@ echo Entering "$(cd "$(dirname "$0")" && pwd -P)/$(basename "$0")" in `pwd`
 set -e
 
 export SHELLOPTS
-
-. ./.ci-build-without-test.sh
```

Comment:
Shouldn't this line be moved to `.ci-build-without-test.sh`?

Reply:
It says it's empty, maybe because I didn't change the file content but only the permission code, from 100644 → 100755. If you click the right bottom to view the file, the code is there.